package.json

```diff
@@ -39,7 +39,7 @@
     "ioredis": "^5.3.2",
     "pino": "^8.15.3",
     "pino-pretty": "^10.2.0",
-    "stellar-sdk": "11.0.0-beta.6",
+    "stellar-sdk": "11.0.1",
```

Comment:
Can you actually upgrade to 11.1.0 please? I don't think it'll matter much to your code, but 11.1.0 is the latest. How quickly these things change lol

Reply:
Oh yup, I just saw this version as the recommended version in the release docs but will upgrade. Thanks!